modules/mailer/mailer.go

```diff
@@ -42,7 +42,7 @@ func NewMessageFrom(to []string, from, subject, body string) *Message {
 
 	plainBody, err := html2text.FromString(body)
 	if err != nil || setting.MailService.SendAsPlainText {
-		if strings.Contains(base.TruncateString(body, 100), "<html>"){
+		if strings.Contains(base.TruncateString(body, 100), "<html>") {
```

Comment:
Please revert a885c7f since theire is already a PR for this: #2490

Reply:
Good catch